Changelog.md

```diff
@@ -9,6 +9,7 @@ Compiler Features:
 
 Bugfixes:
  * Code Generator: Avoid including references to the deployed label of referenced functions if they are called right away.
+ * Assembler: De-duplicate subassembly bytecode where possible.
```

Comment:
You know what, this is not exactly in alphabetical order, but I've just noticed that the SMTChecker entries need to be reordered as well, so will do that in a separate PR later anyway.

Reply:
I'd actually not even have called this "de-duplicating"... what we do is we "Avoid duplication" really :-). But fine with it anyways.

Reply:
Since you now rephrased it you could have moved it up as well while you're at it ;-P.

Reply:
Future generations of developers looking to blame us for this change will be furious :-). (Still fine with keeping it out of order)

Reply:
Wait, don't we have to re-order the Changelog before the release anyway (i.e. in alphabetical order)?

Reply:
Only because we're too stupid to keep it alphabetic earlier :-). It *should* be kept alphabetic at all times and the only disordering should stem from merge conflicts. We actually talked about this in the project management channel a while back even I think :-). But it's not a big deal either way.